crates/indexer/src/runtime.rs

```diff
@@ -146,25 +148,28 @@ pub async fn run_forever(config: IndexerConfig, context: Arc<Context>) {
         tailer.run_migrations();
     }
 
+    let starting_version_from_db = tailer
```

Comment:
by removing this from the `match` below, this is called every time now: even when we proceed to discard it right after, in the case `config.starting_version` is `Some`. Maybe put it back in?

Reply:
Yes but it's just called once per startup and I actually want to show log this.

Reply:
Low cost, possibly prevent huge cost of starting at the wrong place.

Reply:
btw i added this intentionally when i find myself wondering all the time why we started at X instead of Y.